rotkehlchen/chain/ethereum/modules/aave/aave.py

```diff
@@ -174,6 +207,41 @@ def _get_balances(
 
             aave_balances[account] = AaveBalances(lending=lending_map, borrowing=borrowing_map)  # type: ignore
 
+        for account, balance_sheet in (  # for v3
+            given_eth_balances if isinstance(given_eth_balances, dict) else given_eth_balances()
+        ).items():
+            for asset, balance in balance_sheet.assets.items():
+                if (
+                    asset.is_evm_token() and
+                    (asset := asset.resolve_to_evm_token()).protocol == CPT_AAVE_V3 and
+                    (reserve_data := self._get_v3_reserve_data(asset)) is not None
+                ):
+                    aave_balances[account].lending[CryptoAsset(
+                        # existence of underlying_tokens is checked in _get_v3_reserve_data
+                        strethaddress_to_identifier(asset.underlying_tokens[0].address),  # type: ignore[attr-defined]  # asset is resolved to evm token
+                    )] = AaveLendingBalance(
+                        balance=balance,
+                        # converting to APY according to https://docs.aave.com/developers/guides/rates-guide#formatting-rates  # noqa: E501
+                        apy=(ONE + reserve_data.liquidity_rate / YEAR_IN_SECONDS) ** YEAR_IN_SECONDS - ONE,  # noqa: E501
+                        version=3,
+                    )
+
+            for liability, balance in balance_sheet.liabilities.items():
+                if (
+                    liability.is_evm_token() and
+                    (liability := liability.resolve_to_evm_token()).protocol == CPT_AAVE_V3 and
+                    (reserve_data := self._get_v3_reserve_data(liability)) is not None
+                ):
+                    aave_balances[account].borrowing[CryptoAsset(
+                        # existence of underlying_tokens is checked in _get_v3_reserve_data
+                        strethaddress_to_identifier(liability.underlying_tokens[0].address),  # type: ignore[attr-defined]  # asset is resolved to evm token
+                    )] = AaveBorrowingBalance(
+                        balance=balance,
+                        variable_apr=reserve_data.variable_borrow_rate,
+                        stable_apr=reserve_data.stable_borrow_rate,
+                        version=3,
+                    )
+
```

Comment:
shouldn't you clean the cache? The apy will be different between calls I believe and as it is now you will need to close the app to get updated values